core/src/UserSettingsEditor.html

```diff
@@ -180,6 +181,7 @@
                   <input
                     type="text"
                     class="fd-input fd-input-group__input"
+                    placeholder="{schemaItem.placeholder && schemaItem.placeholder}"
```

Comment:
Why do you check it twice?

Reply:
Cause first one check is is exist and second one add value